isolate.sh

```diff
@@ -73,10 +73,10 @@ function main() {
     prereq
     local ns_list=$(gather_csmaps_ns)
     pause
```

Comment:
the pause function calls removeNSS, but we should not be deleting NSS CRs so early. The removeNSS should be run after cs-maps handling